Wrappers/Python/examples/ui_examples/SingleViewerCentralWidget2.py

```diff
@@ -11,13 +11,20 @@ class SingleViewerCenterWidget(QtWidgets.QMainWindow):
 
     def __init__(self, parent=None, viewer=viewer2D):
         QtWidgets.QMainWindow.__init__(self, parent)
-        self.setGeometry(450, 250, 1000, 1000)
-        self.frame = QCILViewerWidget(parent, viewer=viewer, shape=(2000, 2000))
+        x,y = 450, 250
+        dx, dy = 1000, 1000
```

Comment:
Notice that the geometry of a Qt window is handled by Qt not by VTK, so `shape` is useless.